strapi/strapi
# [Blocks editor] Images too big in large viewports

### What does it do?

Fix the dimensions and ratio of big images uploaded in the Blocks editor

### Why is it needed?

Because with big images we have too much space taken in the editor

### How to test it?

Upload images with large width or height in the Blocks editor and check the rendering
